src/DataViews/include/DataViews/LiveFunctionsDataView.h

```diff
@@ -62,6 +63,7 @@ class LiveFunctionsDataView : public DataView {
       const orbit_grpc_protos::InstrumentedFunction& instrumented_function);
 
   absl::flat_hash_map<uint64_t, orbit_client_protos::FunctionInfo> functions_{};
+  absl::flat_hash_map<uint64_t, std::vector<uint64_t>> timer_durations_;
```

Comment:
As I told you yesterday, this is creating technical debt as we are storing a lot of data inside a DataView and is going in the opposite direction of definition we have taken in the past.

I would prefer to NOT show a histogram if they have more than 1 million timers.

Let's discuss it offline, but in any case we should add a TODO (you can link this bug http://b/191333567).
Example: 
TODO (b/191333567): We shouldn't store this data inside the data views.

Reply:
Similar here, add a TODO before resolving the comment.

Reply:
I think, only one todo should suffice. I've clicked resolved after I've written it. Probably, should have done that after committing. Anyway, now it's committed.

Reply:
Nit: TODO(b/123) ([go/todo-style](https://goto.google.com/todo-style))